sys-apps/portage/portage-9999.ebuild

```diff
@@ -28,13 +26,16 @@ fi
 LICENSE="GPL-2"
 SLOT="0"
 IUSE="apidoc build doc gentoo-dev +ipc +native-extensions +rsync-verify selinux test xattr"
+REQUIRED_USE="${PYTHON_REQUIRED_USE}"
 RESTRICT="!test? ( test )"
 
 BDEPEND="
-	dev-python/setuptools[${PYTHON_USEDEP}]
+	${PYTHON_DEPS}
+	python_targets_python3_12? ( dev-python/setuptools[${PYTHON_USEDEP}] )
```

Comment:
This can go, I think?

Reply:
Haven't tried but made me think of libmbim's https://bugs.gentoo.org/911418#c5 and currently I see:
```
py_mod = import('python')
# TODO: Add "pure : false" here instead of py.install_sources() when requiring Meson >=0.64.0.
py = py_mod.find_installation()
```
So find_installation is going to fail without distutils I think

Reply:
See mesonbuild/meson#7702. Installing setuptools is a workaround. We may need this in other packages until this is fixed, but loads of other things pull in setuptools anyway.

Reply:
Could you pop in a comment above BDEPEND just noting that? Thanks!

Reply:
Done.